lib/azureServiceClient.ts

```diff
@@ -115,67 +87,81 @@ export class AzureServiceClient extends msRest.ServiceClient {
    * @returns {Promise<msRest.HttpOperationResponse>} result - The final response after polling is complete.
    */
   async getLongRunningOperationResult(resultOfInitialRequest: msRest.HttpOperationResponse, options?: msRest.RequestOptionsBase): Promise<msRest.HttpOperationResponse> {
```

Comment:
Did you mean to remove the `async` here?

Reply:
I think I kept it in here because refactoring a while loop that contained an `await` looked like a hassle-- will maybe save it for the work item that needs to change LRO behavior anyway?